src/fetch/data_preprocessing.py

```diff
@@ -0,0 +1,34 @@
+"""Preprocessing of data for payments
+Accounting period starting on: 2024-04-30"""
+
+from pandas import DataFrame
+
+
+def preprocess_batch_data(batch_data_df: DataFrame) -> DataFrame:
+    """Preprocess batch data"""
+    return batch_data_df
+
+
+def preprocess_trade_data(trade_data_df: DataFrame) -> DataFrame:
+    """Preprocess trade data"""
```

Comment:
Maybe add some more comments here to provide context. Such as

`this is the place to add any manual fixes due to erroneous prices etc`

Reply:
I added some disclaimer in the doc string at the top and a short description of the processing in the function itself.